.ci/releasePipeline.groovy

```diff
@@ -174,7 +174,7 @@ void signWinBinaries(Map args = [source: null, version: null, destDir: null, pro
 
 void createShaSum(Map args = [source: null, dest: null]) {
     stage('Compute SHA sum') {
-        sh "sha256sum ${args.source} | sed 's, .*/, ,' > ${args.dest}"
+        sh "sha256sum ${args.source} | sed 's, .*/,  ,' > ${args.dest}"
```

Comment:
Why is this double space needed here? What was the exact problem and why does this fix it?
The PR description is lacking in why it was broken and why this space fixes it. I have no clue.

Reply:
c/p from https://unix.stackexchange.com/a/310401/481532
Binary mode is indicated with ‘*’, text mode with ‘ ’ (space).

First space is separator, second indicates that it was read in text mode. Probably the binary mode would be better, but this works well